src/v/raft/tests/membership_test.cc

```diff
@@ -56,32 +56,37 @@ FIXTURE_TEST(add_two_nodes_to_the_cluster, raft_test_fixture) {
     BOOST_REQUIRE_EQUAL(leader.consensus->config().brokers().size(), 3);
 };
 
-void verify_node_is_behind(raft_group& gr, model::node_id removed) {
-    tests::cooperative_spin_wait_with_timeout(3s, [&gr, removed] {
-        return ss::async([&gr, removed] {
+/**
+ * After removing a node, check that the remaining nodes
+ * have all advanced past it, to the same offset.
+ *
+ * @param removed_offset The offset of the just-removed group.
+ */
+void verify_removed_node_is_behind(
+  raft_group& gr, model::offset removed_offset) {
+    tests::cooperative_spin_wait_with_timeout(3s, [&gr, removed_offset] {
+        return ss::async([&gr, removed_offset] {
```

Comment:
hmm, looks like `removed_offset` is unused?

Reply:
Oops!  Fixed.